src/views/News/index.js

```diff
@@ -43,465 +70,556 @@ const styles = {
-                        onChange={this.onChange.bind(this)}
-                        // helperText="Please choose a category."
-                        style={{ minWidth: '7rem' }}
-                      >
-                        {this.state.categories.map((category) => (
-                          <MenuItem key={category.categoryID} value={category.categoryID}>
-                            {category.categoryName}
-                          </MenuItem>
-                        ))}
-                      </TextField>
-                    </Grid>
-
-                    {/* SUBJECT ENTRY */}
-                    <Grid item xs={12}>
-                      <TextField
-                        label="Subject"
-                        margin="dense"
-                        fullWidth
-                        name="newPostSubject"
-                        type="search"
-                        value={this.state.newPostSubject}
-                        onChange={this.onChange.bind(this)}
-                        // helperText="Please enter a subject."
-                      />
-                    </Grid>
-
-                    {/* BODY ENTRY */}
-                    <Grid item xs={12}>
-                      <TextField
-                        label="Body"
-                        margin="normal"
-                        multiline
-                        fullWidth
-                        rows={4}
-                        variant="outlined"
-                        name="newPostBody"
-                        value={this.state.newPostBody}
-                        onChange={this.onChange.bind(this)}
-                        // helperText="Please enter a body."
-                      />
-                    </Grid>
-
-                    {/* Image dropzone will be added here */}
-                    {/* <Grid item xs={12}>
-                          <Dropzone></Dropzone>
-                        </Grid> */}
-                    <Grid item>
-                      {/* SUBMISSION GUIDELINES */}
-                      <Typography variant="caption" color="textSecondary" display="block">
-                        Student News is intended for announcing Gordon sponsored events, lost and
-                        found, rides, etc. All submissions must follow the Student News guidelines
-                        and will be reviewed at the discretion of The Office of Student Life...
-                        <a href="https://gordonedu.sharepoint.com/:b:/g/StudentLife/admin/EY22_o3g6vFEsfT2nYY-8JwB34OlYmA1oaE1f4FTGD2gew">
-                          More Details
-                        </a>
-                      </Typography>
-                    </Grid>
+                  {/* IMAGE ENTRY */}
+                  <Grid item xs={12}>
+                    <div className="gc360-photo-dialog-box">
+                      <DialogContent className="gc360-photo-dialog-box_content">
+                        <DialogContentText className="gc360-photo-dialog-box_content_text">
```

Comment:
It's confusing to have DialogContent and DialogContentText wrapping the image uploader when it isn't a dialog (but is inside the DialogContent of the Post News dialog).

Reply:
I tried removing all those tags. The format becomes ugly when they're removed.

Reply:
That probably has more to do with the classes applied to them, but I'm not sure it's worth the time to sort that out. This doesn't matter all that much anyways.